src/UI/pages/PrivateRoutes/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { FC } from "react";
+
+import { Outlet, Navigate } from "react-router-dom";
+import ROUTES from "../../../constants/routes";
+
+const isAuthorized: boolean = false;
+
+export const PrivateRoutes: FC = () => {
```

Comment:
rename as it is not obsious what it should do 
better AuthRoutes and NonAuthRoutes

Reply:
AuthRoute and NonAuthRoute